db/migrate/20210126171415_rename_mentor_to_owner.rb

```diff
@@ -0,0 +1,5 @@
+class RenameMentorToOwner < ActiveRecord::Migration[6.0]
+  def change
+  	rename_column :groups, :mentor_id, :owner_id
```

Comment:
@tachyons I think its better to not change schema unnecessarily. It shouldn't matter much as our DB is small but as a practice I felt its better to not rename. We can have a note that this is not mentor but actually group admin/owner but for schema is the same. Please comment.

Reply:
For zero downtime migration, code new field can be added with the name owner_id, store data in both attributes and later remove mentor_id once application code stop referencing that attribute. But it makes things unnecessarily complicate things in our scale

Reply:
I was thinking of letting the owners be called mentors, and we name the TA level privillege to something else like 'moderator'. How does that sound?

Reply:
Moderator is a term used in a community setting, so its not very apt. Teaching Assistant sounds good. 

CC: @Shivansh2407 @sal2701

Reply:
Yeah! Teaching assistant sounds good to me as well 👍

Reply:
Teaching assistant option can be ambiguous as this option may be for a teacher, mentor, guide depends on the context

Reply:
How about just 'assistant' then?

Reply:
We can go with primary mentor and mentor. We need to get this PR merged as early as possible

Reply:
But that would still require the schema change right?

Reply:
Yes, but not an issue at the moment

Reply:
In that case, I feel like owner seems more explicit? What do you think?

Reply:
I had a discussion with @satu0king, and we agreed upon primary_mentor

Reply:
Okay, I will make the changes then :+1: